src/util/xml_to_json.cc

```diff
@@ -113,10 +114,22 @@ std::string Xml2Json::getAsString(const char* str)
     return "\"" + escape(str, '\\', '"') + '"';
 }
 
-std::string Xml2Json::getValue(const char* text)
+std::string Xml2Json::getValue(const char* name, const char* text, const Hints& hints)
 {
-    // type information is gone, we need to guess...
     std::string str = text;
+    auto hintsType = hints.asType;
```

Comment:
This makes a copy ☺️ shouldn't be needed here.

Reply:
Well I just copy what PR #882 did. Changed it to reference